frontend/src/main.ts

```diff
@@ -135,6 +135,14 @@ function init(): void {
   });
 
   router.trigger("page-loaded");
+
+
+  const storedTheme = localStorage.getItem("theme") ?? "auto";
```

Comment:
I don't think this logic here is needed, adding / removing the attribute should all happen in the `store.subscribe` (but actually attaching that subscriber should happen here)